addons/l10n_es/data/account.account.template-coop-full.csv

```diff
@@ -0,0 +1,183 @@
+"account_coop_full_120","Remanente",120,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_121","Resultados negativos de ejercicios anteriores",121,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_129","Resultado del ejercicio",129,"account.data_unaffected_earnings","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_130","Subvenciones oficiales de capital",130,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_132","Otras subvenciones, donaciones y legados",132,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_133","Ajustes por valoración en activos financieros disponibles para la venta",133,"account.data_account_type_equity","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_1340","Cobertura de flujos de efectivo",1340,"account.data_account_type_equity","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_1341","Cobertura de una inversión neta en un negocio en el extranjero",1341,"account.data_account_type_equity","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_135","Diferencias de conversión",135,"account.data_account_type_equity","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_136","Ajustes por valoración de activos no corrientes y grupos enajenables de elementos, mantenidos para la venta",136,"account.data_account_type_equity","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_140","Provisión por retribuciones del personal",140,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","True"
+"account_coop_full_146","Provisión para reestructuraciones",146,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","True"
+"account_coop_full_147","Provisión por transacciones con pagos basados en instrumentos de patrimonio",147,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","True"
+"account_coop_full_148","Fondo de Educación, Formación y Promoción a largo plazo",148,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_1500","Capital Social Cooperativo a largo plazo considerado pasivo financiero aportaciones obligatorias",1500,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
+"account_coop_full_1501","Capital Social Cooperativo a largo plazo considerado pasivo financiero aportaciones voluntarias",1501,"account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_coop_full","False"
```

Comment:
Both cases can happen in cooperatives, the PYMEs and the full. That inheritance chain you propose can help to alleviate the duplicated accounts.

Reply:
@jco-odoo There are some accounts in common that is not for cooperatives. So I moved from common to the specific Chart of Accounts of PYME, Full and Assoc.
 
The account are this:
"account_common_112","Reserva legal","112","account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_common","False"
"account_common_113","Reservas voluntarias","113","account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_common","False"
"account_common_150","Acciones o participaciones consideradas como pasivos financieros","150","account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_common","False"
"account_common_502","Acciones o participaciones a corto plazo consideradas como pasivos financieros","502","account.data_account_type_current_liabilities","l10n_es.account_chart_template_common","False"
"account_common_521","Deudas a corto plazo","521","account.data_account_type_current_liabilities","l10n_es.account_chart_template_common","False"
"account_common_526","Dividendo activo a pagar","526","account.data_account_type_payable","l10n_es.account_chart_template_common","True"
"account_common_5585","Socios por desembolsos exigidos sobre acciones o participaciones consideradas como pasivos financieros","5585","account.data_account_type_non_current_liabilities","l10n_es.account_chart_template_common","False"

The separation in es_common -> es_coop_minimal -> es_coop_full or es_coop_pyme, can be done, is not complicated.

Reply:
@acysos I have done that in the https://github.com/odoo/odoo/pull/126694 PR.  Just to avoid having the es_coop_common and double accounts, I saw that 2935 is like double and 663 and 796 are split in the coop_full compared to the coop_pymes.   I wonder if https://github.com/odoo/odoo/pull/126694/files#diff-5a18f30c03f7e0ceaf5bd9516e95f044fbce0087c3ee0928a5591ca7a662129aR53 is a good solution for this to you as we are increasing the numbers of certain accounts.

Reply:
@acysos any news about the last comment?